angularjs/angular.d.ts

```diff
@@ -1607,7 +1604,7 @@ declare module angular {
 
     interface IDirective {
         compile?: IDirectiveCompileFn;
-        controller?: any;
+        controller?: {};
```

Comment:
You are correct. It's needed only in the declaration of IDirectiveLinkFn. It makes no sense here. I will change it back.